docs/install/quick-start-guide.md

```diff
@@ -0,0 +1,148 @@
+storageclass.storage.k8s.io/local-path   rancher.io/local-path   Delete          WaitForFirstConsumer   false                  3h10m
+```
+
+## Step 3 - Create the StorageOS namespace
+
+```bash
+kubectl create namespace storageos
+```
+
+## Step 4 - Customise and install the helm chart
+
+We make a few changes to the default helm chart values, so we can run in smaller non-production sized clusters.
+
+> ⚠️ Make sure to set the values of `ONDAT_USERNAME` and `ONDAT_PASSWORD`
+
+```bash
+export ONDAT_USERNAME="changeme"
+export ONDAT_PASSWORD="changeme"
+
+helm install ondat ondat/ondat --namespace storageos \
+--set ondat-operator.cluster.admin.username=${ONDAT_USERNAME},\
+ondat-operator.cluster.admin.password=${ONDAT_PASSWORD},\
+etcd-cluster-operator.cluster.namespace=storageos-etcd,\
+etcd-cluster-operator.cluster.replicas=3,\
+etcd-cluster-operator.cluster.storageclass=local-path,\
+etcd-cluster-operator.cluster.storage=6Gi,\
+etcd-cluster-operator.cluster.resources.requests.cpu=100m,\
+etcd-cluster-operator.cluster.resources.requests.memory=300Mi
+```
+
+## Step 5 - Wait until the storageos storage class has been created
+
+It can take a few seconds for the storageos storage class to be created. We need to wait for this before we can set it as the default storage class.
+
+The following command can be used to ensure the storageos storage class was successfully deployed. The expected output is also shown
+
+```bash
+> kubectl -n storageos get storageclass -w
+NAME         PROVISIONER             RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
+local-path   rancher.io/local-path   Delete          WaitForFirstConsumer   false                  3h4m
+storageos    csi.storageos.com       Delete          Immediate              true                   75m
+```
+
+## Step 6 - Set the default storage class
+
+Set the storageos storage class as the default
+
+```bash
+kubectl patch storageclass local-path -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"false"}}}'
+kubectl patch storageclass storageos -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'
+```
+
+## Step 7 - Ensure everything is running
+
+It can take a couple of minutes for the cluster to converge. A ready cluster will look like this:
+
+```bash
+> kubectl -n storageos get pods,storageclass,svc                    
+NAME                                           READY   STATUS    RESTARTS      AGE
+pod/ondat-controller-manager-5f7f669b4-fqj57   1/1     Running   0             73m
+pod/ondat-controller-manager-5f7f669b4-jmbfk   1/1     Running   0             73m
+pod/ondat-ondat-operator-675c4c9c88-9jgrv      2/2     Running   0             73m
+pod/ondat-proxy-6887466cd5-vlkq6               1/1     Running   0             73m
+pod/storageos-api-manager-6677c95579-qdght     1/1     Running   0             71m
+pod/storageos-api-manager-6677c95579-rzwqr     1/1     Running   1 (71m ago)   71m
```

Comment:
You can make it look better if you remove all the restarts 👼

Reply:
I agree, but users shouldn't be scared if they see restarts, maybe this conveys that, but I'm happy to change if it doesn't